plugwise_usb/nodes/circle.py

```diff
@@ -418,6 +428,9 @@ def _collect_energy_pulses(self, start_utc: datetime, end_utc: datetime):
                     str(self._energy_last_populated_slot),
                     str(self._energy_last_collected_timestamp),
                 )
+                if _mem_address == self._last_log_address:
+                    self._last_log_address_failed = True
+                _energy_history_failed = True
```

Comment:
This line needs 1 ident less?
So that it lines up with the above `else`?

Reply:
I mean line 433

Reply:
no it is correct, 433 is executed inside the else of line 420.

Reply:
See also my other feedback on Discord.